extensions/ql-vscode/src/remote-queries/view/FileCodeSnippet.tsx

```diff
@@ -188,36 +187,36 @@ const FileCodeSnippet = ({
       <CodeContainer>
         {code.map((line, index) => (
           <div key={index}>
-            <Box display="flex">
-              <Box
-                p={2}
-                borderStyle="none"
-                paddingTop="0.01em"
-                paddingLeft="0.5em"
-                paddingRight="0.5em"
-                paddingBottom="0.2em">
+            <div style={{ display: 'flex' }} >
+              <div style={{
+                borderStyle: 'none',
+                paddingTop: '0.01em',
+                paddingLeft: '0.5em',
+                paddingRight: '0.5em',
+                paddingBottom: '0.2em'
+              }}>
                 {startingLine + index}
-              </Box>
-              <Box
-                flexGrow={1}
-                p={2}
-                borderStyle="none"
-                paddingTop="0.01em"
-                paddingLeft="1.5em"
-                paddingRight="0.5em"
-                paddingBottom="0.2em"
-                sx={{ wordBreak: 'break-word' }}>
+              </div>
+              <div style={{
+                flexGrow: 1,
+                borderStyle: 'none',
+                paddingTop: '0.01em',
+                paddingLeft: '1.5em',
+                paddingRight: '0.5em',
+                paddingBottom: '0.2em',
+                wordBreak: 'break-word'
+              }}>
                 <CodeLine
                   line={line}
                   lineNumber={startingLine + index}
                   highlightedRegion={highlightedRegion} />
-              </Box>
-            </Box>
+              </div>
+            </div>
             {message && severity && <Message
               message={message}
               currentLineNumber={startingLine + index}
               highlightedRegion={highlightedRegion}
-              borderColor={getSeverityColor(severity)}>
+              borderLeftColor={getSeverityColor(severity)}>
```

Comment:
This looks like a difference from the old implementation. Is this intentional?

Reply:
Yes it is. I've updated the prop for the `Message` component to be `borderLeftColor` instead of `borderColor` to avoid ambiguity with the variable called `bordedColor`.